app/src/main/java/com/example/keybindhelper/dao/CurrentProjectManager.java

```diff
@@ -3,12 +3,9 @@
 
 import androidx.lifecycle.MutableLiveData;
 
-import com.example.keybindhelper.dto.Group;
```

Comment:
Removed multiple unused import statements with Ctrl+Alt+O
source: https://kotlinlang.org/spec/packages-and-imports.html